libnavigation-util/build.gradle

```diff
@@ -1,8 +1,22 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
+apply plugin: 'org.jetbrains.dokka'
 apply from: "${rootDir}/gradle/ktlint.gradle"
 
+dokka {
+    outputDirectory = "$buildDir/javadoc"
```

Comment:
do we really need _docs_ for `util` module? we use it as an internal dependency only

Reply:
We have to declare a `dokka` job in the `utils` module, otherwise, the release script is going to fail because it depends on it:
https://github.com/mapbox/mapbox-navigation-android/pull/2427/files#diff-6e0a22179ff8811fdb959c72e981af01R92-R94

But as you can see, the whole package is excluded so there's no doc jar generated for this module:
https://github.com/mapbox/mapbox-navigation-android/pull/2427/files#diff-010b612c4432835f537410d8531b5b1fR13-R16

Reply:
thanks for clarification, didn't note gradle' task 👍 great work @LukasPaczos